### PR TITLE
World save to disk

### PR DESCRIPTION
**Pull request by huntears**

### CODE REVIEW

File: cubic-server/world_storage/ChunkColumn.cpp [hunk 1]
```diff
@@ -606,4 +609,77 @@ void ChunkColumn::processRandomTick(uint32_t rts)
     }
 }
 
+nbt_tag_t *ChunkColumn::toRegionCompatibleFormat()
+{
+    int32_t dataVersion = 2865; // TODO(huntears): This sucks, no idea what that value is
```
Comment (Trompettesib) [inline cubic-server/world_storage/ChunkColumn.cpp:614]: image
I have this on a 1.19.3 save if you want

Comment (huntears) [reply cubic-server/world_storage/ChunkColumn.cpp:614]: tbh i don't think it really matters that much but I will update it.

**Review by STMiki — APPROVED** [review]
¯\\\_(ツ)_/¯

**Review by Trompettesib — CHANGES_REQUESTED** [review]
Overall the code is great, but there are some compilation warnings and a crash. See screenshots:
image
image
image
image

**Review by Trompettesib — APPROVED** [review]
:rocket:

**Review by NekoSelena — APPROVED** [review]

**Pull request merged**